<dec f='include/c++/5/exception' l='102' type='void std::terminate()'/>
<use f='include/c++/5/bits/nested_exception.h' l='74' u='c' c='_ZNKSt16nested_exception14rethrow_nestedEv'/>
<doc f='include/c++/5/exception' l='100'>/** The runtime will call this function if %exception handling must be
   *  abandoned for any reason.  It can also be called by the user.  */</doc>
<use f='include/c++/5/thread' l='151' u='c' c='_ZNSt6threadD1Ev'/>
<use f='include/c++/5/thread' l='159' u='c' c='_ZNSt6threadaSEOS_'/>
